### ISSUE TITLE
ApolloError: Cannot read properties of undefined (reading 'map')

### ISSUE THREAD

**Issue by AndreTheHunter** [apollographql/apollo-client-devtools#1524]
Error on Queries tab:

```
ApolloError: Cannot read properties of undefined (reading 'map')

ApolloError
    at new t (chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:339389)
    at chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:377644
    at i (chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:347140)
    at chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:347054
    at new Promise (<anonymous>)
    at Object.then (chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:347020)
    at Object.next (chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:347150)
    at st (chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:259259)
    at lt (chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:259772)
    at t.next (chrome-extension://jdkknkkbebbapilgoeccciglkfbmbnfm/panel.js:1:260312)
```

### `@apollo/client` version
3.8.7

### Apollo Client Devtools version
4.18.6

**Comment by jan-bednar** [comment]
Im seeing the same thing with 3.7.0 and 4.18.6 versions

**Comment by jan-bednar** [comment]
I just noticed the error shows when one of the queries on page ends loading with error response.

**Comment by jerelmiller** [comment]
Thanks for the report! We'll try and look into this soon. Thanks for the context @jan-bednar! That should help narrow it down.

**Comment by jan-bednar** [comment]
https://github.com/user-attachments/assets/cd0f3c4b-c251-4109-8a3c-b91d3666e64e

Here is the video of the interaction. I can reproduce this with 100% on our site. The video starts with page refresh (0 queries) and once it loads the queries show up but then one query fails and we get the error shown. When then switching tabs its also shows up for a second, before the error overlay is presented

**Comment by jerelmiller** [comment]
I've got #1538 up that I'm hoping fixes this. Its the only place that I could find uses `.map` related to errors since I can't seem to reproduce this myself.

I'll try and get this released this afternoon. I will keep you posted!

**Comment by jerelmiller** [comment]
I just published [4.18.9](https://github.com/apollographql/apollo-client-devtools/releases/tag/v4.18.9) which should go live in the extension stores within the next couple hours. Please let me know if you're still experiencing this issue after upgrading and I'd be happy to reopen this. Thanks!
